core/src/main/java/org/apache/accumulo/core/client/rfile/RFileSummariesRetriever.java

```diff
@@ -90,7 +90,7 @@ public SummaryOptions endRow(Text endRow) {
       for (RFileSource source : sources) {
         SummaryReader fileSummary = SummaryReader.load(conf, acuconf, source.getInputStream(),
             source.getLength(), summarySelector, factory,
-            CryptoServiceFactory.getConfigured(acuconf));
+            CryptoServiceFactory.newInstance(acuconf));
```

Comment:
`acuconf` is default configuration. you should pass in your crypto configuration

Reply:
We are in the same boat here as RFileScanner

Reply:
Nevermind, crypto won't actually work with the summaries API as-is.

Reply:
Opened #612 to fix this.